snyk-operator/build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/operator-framework/helm-operator:v0.15.1
+FROM quay.io/operator-framework/helm-operator:v1.1.0
```

Comment:
I'm curious if is it possible it break any integration test or the deployment process to create the Operator? I think it should not... however...

Reply:
Let's find out 🙂 It's all good to revert until we publish to GitHub so we have plenty of time to detect issues